wikimedia/wikipedia-ios
# Allow tapping cell to expand and collapse talk page cells

**Phabricator:** https://phabricator.wikimedia.org/T310279 (partially)

### Notes
Allow tapping a talk page cell to expand/collapse the conversation.

From Phabricator ticket:
> Topics found on each article talk page will be collapsed when first opened. Contributor/reader can see comments once they tap anywhere on the topic cell.

### Test Steps
1. Confirm selecting cells expands/collapses the conversation
2. Confirm tapping the disclosure button still expands/collapses cell
3. Confirm tapping subscribe/unsubscribe button stills work and prevents expanding/collapsing
4. Confirm tapping the reply buttons blocks the cells from expanding/collapsing
